### PR TITLE
adding more info for scripts

### PR DESCRIPTION
**Pull request by g-m-taylor**
Adding some useful commands

### CODE REVIEW

File: ad-hoc-scripts/encyprt_nin_dob.sql [hunk 1]
```diff
@@ -2,7 +2,40 @@ CREATE EXTENSION IF NOT EXISTS pgcrypto;
...
+'::PUBLICKEY'
+,'base64'),'UTF8'))));
 
 update cqc."Worker"
-set "NationalInsuranceNumberEncryptedValue" = armor(pgp_pub_encrypt("NationalInsuranceNumberValue" :: text, dearmor(convert_from(decode('::PUBLIC KEY','base64'),'UTF8'))));
+set "NationalInsuranceNumberEncryptedValue" = armor(pgp_pub_encrypt("NationalInsuranceNumberValue" :: text, dearmor(convert_from(decode(
+'::PUBLIC KEY'
+,'base64'),'UTF8'))));
+
+-- OR BOTH TOGETHER
```
Comment (ross-mcfadyen) [inline ad-hoc-scripts/encyprt_nin_dob.sql:14]: Is it worth choosing which approach to go for so the script itself can be run?

Also typo in filename 😄

Comment (ross-mcfadyen) [reply ad-hoc-scripts/encyprt_nin_dob.sql:14]: Doing them together is probably cleaner but either or 🤷

Comment (g-m-taylor) [reply ad-hoc-scripts/encyprt_nin_dob.sql:14]: I just had to google where the typo is 🙈 ah dyslexia.       Running them together is faster but I have had to run them separately as well (when debugging etc) so probably best to keep both in, but Il write a note that running them together is better when deploying.

Comment (ross-mcfadyen) [reply ad-hoc-scripts/encyprt_nin_dob.sql:14]: Ah! Sorry bud, I should've pointed it out.

OK, cool. Sounds good 👍

**Review by ross-mcfadyen — APPROVED** [review]

**Pull request merged**